packages/cli/test/functional/test_site/site.json

```diff
@@ -206,7 +206,7 @@
       "pre": "\n\n# Markbind Plugin Pre-render",
       "post": "<p>Node Modules Plugin Post-render</p>"
     },
-    "filterTags": {
+    "filterTacgs": {
```

Comment:
?

Reply:
Sorry, this must have been a typo. Thanks for pointing this out!